pkg/daemon/update.go

```diff
@@ -48,6 +59,64 @@ func (dn *Daemon) update(oldConfig, newConfig *mcfgv1.MachineConfig) error {
 	return dn.reboot()
 }
 
+// reconcilable checks the configs to make sure that the only changes requested
+// are ones we know how to do in-place. if we can't do it in place, the node is
+// marked as degraded.
+//
+// we can only update machine configs that have changes to the files,
+// directories, links, and systemd units sections of the included ignition
+// config currently.
+func (dn *Daemon) reconcilable(oldConfig, newConfig *mcfgv1.MachineConfig) (bool, error) {
+	oldIgn := oldConfig.Spec.Config
+	newIgn := newConfig.Spec.Config
+
+	// Ignition section
```

Comment:
Thank you for using this style of comments. This is wonderful :rainbow: :heart:.